frontend/kesaseteli/employer/browser-tests/utils/url.utils.ts

```diff
@@ -26,17 +34,33 @@ export const getUrlUtils = (t: TestController) => {
       await t.navigateTo(getEmployerUiUrl(`/company`));
       await pageIsLoaded();
     },
+    async refreshPage() {
+      await refreshPage();
```

Comment:
TBH, I don't know which function this is calling. Is it calling `async refreshPage() {`? Or `const refreshPage = ClientFunction(() => {`?

Having the same name for these functions is a bit misleading.

Reply:
replaced it with ClientFunction straight